lib/tasks/bulk_loaders/load_historic_carbon_intensity_data.rake

```diff
@@ -0,0 +1,10 @@
+namespace :bulk_load do
+  task historic_carbon_intensity_data: [:environment] do
+    pathname = Pathname.new("tmp/uk_carbon_intensity.csv")
+    raise ArgumentError, 'File is missing, should be in "tmp/uk_carbon_intensity.csv"' unless pathname.exist?
+    CSV.foreach(pathname, col_sep: ',') do |row|
+      date = row.shift
+      DataFeeds::CarbonIntensityReading.create(reading_date: date, carbon_intensity_x48: row)
```

Comment:
This uses PH's file - possible we can switch this to just use the 'API' and call it with an after party job. Checking with PH when back